tox.ini

```diff
@@ -1,27 +1,30 @@
 [tox]
-envlist = 
-    py27,py33,py34,py35,pypy,pypy3,coverage,docs
+envlist =
+    py27,py34,py35,py36,pypy,pypy3,coverage,docs
 
 [testenv]
-commands = 
+commands =
     python setup.py -q test -q
+    sphinx-build -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+deps =
+    .[test,docs]
 
 [testenv:coverage]
 usedevelop = true
 basepython =
     python2.7
-commands = 
-    nosetests --with-xunit --with-xcoverage
+commands =
+    coverage run setup.py -q test -q
+    coverage run -a -m sphinx -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+    coverage report
```

Comment:
@jamadden `coverage report` has an option `--fail-under=MIN`: "Exit with a status of 2 if the total coverage is less than MIN." which might help to fail on TravisCI, too.